src/main/resources/lib/s3Plugin/blockWrapper.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define">
+  <!-- TODO remove and switch to div after baseline is 2.264 or newer -->
+  <j:choose>
+    <j:when test="${divBasedFormLayout}">
+      <div>
+        <d:invokeBody/>
+      </div>
+    </j:when>
+    <j:otherwise>
+      <table width="100%" padding="0">
```

Comment:
why padding=0 ?

Reply:
to reduce it's size. I did this without thinking too hard. I can be remove if you prefer.

Reply:
just hadn't seen it before and it wasn't in the original, no issues with it though

Reply:
I saw it on run-condition plugin.